### PR TITLE
chore(e2e): turn on the i18n editview.spec.ts tests

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

Turn on and adjust the i18n e2e tests

### Why is it needed?

To test stuff

### How to test it?

CI tests should pass

### CODE REVIEW

File: tests/e2e/tests/i18n/editview.spec.ts [hunk 1]
```diff
@@ -608,129 +608,129 @@ test.describe('Edit view', () => {
...
+        initialValue,
+        ctbParams: { key: ctbKey, operation: ctbOperation },
+        expectedError,
+      } = validationParams;
+
+      /**
+       * Navigate to our articles list-view where there will be one document already made in the `en` locale
+       */
+      await page.getByRole('link', { name: 'Content Manager' }).click();
+      await clickAndWait(page, page.getByRole('link', { name: 'Article' }));
```
Comment (markkaylor) [inline tests/e2e/tests/i18n/editview.spec.ts:630]: ℹ️ I changed this

File: tests/e2e/tests/i18n/editview.spec.ts [hunk 2]
```diff
@@ -608,129 +608,129 @@ test.describe('Edit view', () => {
...
+      await expect(page.getByRole('combobox', { name: 'Select a locale' })).toHaveText(
+        'English (en)'
+      );
+
+      /**
+       * Attempt to publish through the 'Publish multiple locales' button
+       */
+      await page.getByText('why-i-prefer-football-over-').click();
+      await page.getByRole('button', { name: 'More document actions' }).click();
+      await page.getByRole('menuitem', { name: 'Publish multiple locales', exact: true }).click();
```
Comment (markkaylor) [inline tests/e2e/tests/i18n/editview.spec.ts:723]: ℹ️  I changed this

**Review by markkaylor — COMMENTED** [review]
The diff is a bit confusing. Besides removing the fixme, here are the changes I made

**Review by remidej — APPROVED** [review]
🚂

**Review by remidej — APPROVED** [review]

**Pull request merged**